modal/runner/containers/vllm_unified.py

```diff
@@ -28,27 +27,39 @@ def __init__(
             model_path: str,
             max_model_len: Optional[int] = None,
         ):
-            if num_gpus > 1:
-                # Patch issue from https://github.com/vllm-project/vllm/issues/1116
-                import ray
-
-                ray.shutdown()
-                ray.init(num_gpus=num_gpus, ignore_reinit_error=True)
-
-            super().__init__(
-                VllmParams(
-                    model=model_path,
-                    tensor_parallel_size=num_gpus,
-                    max_model_len=max_model_len,
-                )
+            import sentry_sdk
+
+            sentry_sdk.init(
```

Comment:
we can't init sentry higher up, for other Modal processes too?

Reply:
Think this is as high as we can go to ensure it's run inside the container executing the model. Once we migrate the completions container to a standalone FastAPI app, I'll add it there as well